server/server.js

```diff
@@ -106,6 +106,7 @@ app.use((req, res, next) => {
 		req.headers.host = req.headers.communityhostname;
 	}
 	if (req.hostname.indexOf('localhost') > -1) {
+		req.headers.localhost = req.headers.host;
```

Comment:
Could you explain what this is for? Looks like something for storybook, but I can't totally follow.

Reply:
This is towards making links on the devserver keep you within your devserver rather than leading outbound to `x.pubpub.org`. I could break this out into another PR, since it is a kind of surprising thing to include here.